_includes/navigation.html

```diff
@@ -104,7 +104,7 @@
         {% endif %}
         </div>
 
-        <div class="md:hidden flex-end p-4 text-primary font-display font-bold text-lg"
```

Comment:
I'm scared about removing this class for future conferences (will we remember to add it back?).  Should we add a different color instead?

Reply:
Have now replaced the old text-primary with a new hamburger_menu class (see next commit). That should serve as a reminder + consolidate future text colour changes into navigation.css.